theme.json

```diff
@@ -489,13 +489,6 @@
 					"fontWeight": "500"
 				}
 			},
-			"core/paragraph": {
-				"typography": {
-					"fontSize": "var(--wp--preset--font-size--small)",
-					"letterSpacing": "0.01em",
-					"lineHeight": "1.5"
-				}
-			},
```

Comment:
This shouldn't be necessary. Can you rebase to pull in today's changes?

Reply:
Good call, and rebased. Ty!